website/client/components/groups/groupPlan.vue

```diff
@@ -86,16 +86,13 @@ div
         label(for='new-group-description') {{ $t('description') }}
         textarea.form-control#new-group-description.option-content(cols='3', :placeholder="$t('description')", v-model='newGroup.description')
       .form-group(v-if='type === "guild"')
-        .radio
-          label
-            input(type='radio', name='new-group-privacy', value='private', v-model='newGroup.privacy')
-            | {{ $t('inviteOnly') }}
-
+        .custom-control.custom-radio
+          input.custom-control-input(type="radio", name="new-group-privacy", value="private", v-model="newGroup.privacy")
+          label.custom-control-label {{ $t('inviteOnly') }}
       .form-group
-        .checkbox
-          label
-            input(type='checkbox', v-model='newGroup.leaderOnly.challenges')
-            | {{ $t('leaderOnlyChallenges') }}
+        .custom-control.custom-checkbox
+          input.custom-control-input(type="checkbox", v-model="newGroup.leaderOnly.challenges" :id="_uid")
```

Comment:
what's uid here? using it with `:attr` indicates a Vue variable but I can't find any with this name. Just give it an unique id like `id="create-group-leaderOnlyChallenges-checkbox"`

Reply:
_uid refers to the component's unique id. https://stackoverflow.com/questions/34950867/vue-js-how-to-set-a-unique-id-for-each-component-instance. However looking back on it, putting the component's id on a single element might be a bad idea. I'll change it right now.